Detour/Include/DetourNavMesh.h

```diff
@@ -330,7 +330,7 @@ struct dtNavMeshParams
 	float tileWidth;				///< The width of each tile. (Along the x-axis.)
 	float tileHeight;				///< The height of each tile. (Along the z-axis.)
 	int maxTiles;					///< The maximum number of tiles the navigation mesh can contain.
-	int maxPolys;					///< The maximum number of polygons each tile can contain.
+	int maxPolys;					///< The maximum number of polygons each tile can contain. Does not influence the build step, however adding a tile with more polygons will fail.
```

Comment:
Strictly speaking, Detour does not have a relevant build step, building of tiles is done by Recast.
Maybe something like the following would be clearer?

```suggestion
	int maxPolys;					///< The maximum number of polygons each tile can contain. This and maxTiles are used to calculate how many bits are needed to identify tiles and polygons uniquely.
```

Reply:
Good point. Should it then also be added to `maxTiles`? If the user looks at the HTML documentation, not code...

Reply:
Yes, that would make sense.

Reply:
Added a commit. Let me know if you want me to squash the two instead.

Reply:
No worries, I will squash when merging :)